src/chart/funnel/FunnelSeries.ts

```diff
@@ -48,6 +48,10 @@ type FunnelLabelOption = Omit<SeriesLabelOption, 'position'> & {
         | 'outer' | 'inner' | 'center' | 'rightTop' | 'rightBottom' | 'leftTop' | 'leftBottom'
 };
 
+type FunnelRateLabelOption = Omit<SeriesLabelOption, 'position'> & {
```

Comment:
This commit should be merge in "apache/echarts" repository. why need add this change manually ?  why not pull new updates from original GitHub repository into forked GitHub repository?

Reply:
> This commit should be merge in "apache/echarts" repository
no, it is in next funnel

Reply:
> > This commit should be merge in "apache/echarts" repository
> 
> no, it is in `NextFunnel` branch

please review this change 
@zzn01